packages/cbioportal-frontend-commons/src/components/checkedSelect/BadgeListSelector.tsx

```diff
@@ -0,0 +1,166 @@
+import autobind from 'autobind-decorator';
+import _ from 'lodash';
+import { action, computed, makeObservable } from 'mobx';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+import styles from '../../../../react-mutation-mapper/src/component/mutationMapper/badgeListSelector.module.scss';
+import { BadgeCell } from '../../../../react-mutation-mapper/src/component/filter/BadgeCell';
```

Comment:
This doesn't seem right. `cbioportal-frontend-commons` should not be importing anything from `react-mutation-mapper`